sdcm/cluster.py

```diff
@@ -516,7 +517,10 @@ def _register_grafana_json(json_filename, url):
         json_mapping = {'scylla-data-source.json': 'datasources',
                         'scylla-dash-timeout-metrics.json': 'dashboards/db'}
 
-        process.run('sudo yum install git -y')
+        if platform.linux_distribution()[0].lower() == 'ubuntu':
+            process.run('sudo apt-get --assume-yes install git')
+        else:
+            process.run('sudo yum install git -y')
```

Comment:
We don't have a good support for running sct in ubuntu, it's not enough to just fix this.
This fix should be split to another patch.